btree/btree.c

```diff
@@ -2809,19 +2883,26 @@ static int64_t btree_put_tick(struct m0_sm_op *smop)
 			}*/
 			oi->i_nop.no_node = NULL;
 
-			node_find(&node_slot, &bop->bo_rec.r_key);
+			key_exists = node_find(&node_slot, &bop->bo_rec.r_key);
```

Comment:
you can use `oi->i_node_found` directly instead of `key_exists`.

Reply:
okay